test/singer_clojure/catalog_test.clj

```diff
@@ -0,0 +1,91 @@
+(ns singer-clojure.catalog-test
+  (:require [clojure.test :refer :all]
+            [singer-clojure.catalog :refer :all]))
+
+(deftest get-selected-streams-test
+  (testing "Test that the get-selected-stream properly returns a sequence
+  of selected streams"
+    (let [catalog {"stream_a" { "stream"       "stream_a"
+                               "metadata"      {"selected" true}}
+                   "stream_b" { "stream"       "stream_b"
+                               "metadata"      {"selected" false}}
+                   "stream_c" { "stream"       "stream_c"
+                               "metadata"      {"selected" true}}
+                   "stream_d" { "stream"       "stream_d"
+                               "metadata"      {"selected" false}}
+                   "stream_e" { "stream"       "stream_e"
+                               "metadata"      {"selected" true}}}]
+      (is (= (get-selected-streams catalog)
+             '("stream_a" "stream_c" "stream_e"))))))
+
+(def test-serialized-catalog {"streams"
```

Comment:
Would it benefit to add a comment here that this is used in other tests as well?

Reply:
Fixed